### PR TITLE
FFM-8124: adds ff release notes for 795xx

### PR DESCRIPTION
**Pull request by StrawberryPad**

### CODE REVIEW

**Review by knagurski — DISMISSED** [review]
👍🏻

**Review by laura-mannus — APPROVED** [review]

**Review by laura-mannus — APPROVED** [review]

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -17,20 +17,14 @@ Review the notes below for details about recent changes to Harness Feature Flags
 Harness deploys updates progressively to different Harness SaaS clusters. You can identify the cluster hosting your account in your Account Overview page. The features and fixes in the release notes may not be available in your cluster immediately.
 :::
 
-## Latest - June 5, 2023
+## Latest - June 7, 2023
```
Comment (douglas-j-bothwell) [inline release-notes/feature-flags.md:20]: I assume there's a reason that FF does not include the release # in the headings? I flipped through other modules and they include rel #'s in their headings, ie

Latest - May 29, 2023, version 79505 (CCM)
Latest - May 23, 2023, version 0.12.1 (CE)
etc.

**Comment by StrawberryPad** [comment]
Thanks @laura-mannus !

File: release-notes/feature-flags.md [hunk 2]
```diff
@@ -44,7 +38,15 @@ This release does not include early access features.
   <TabItem value="Fixed issues">
 ```
 
-This release does not include fixed issues.
+#### Feature Flags UI
+
+* In the user onboarding flow, the Create a New Flag button did not behave as expected. This issue has been fixed. (FFM-8019)
+
+* Previously, a user could select Feature Flags in Harness without having a license. Now, users only see the FF module if they have an active license (including a free license). (FFM-8002)
```
Comment (douglas-j-bothwell) [inline release-notes/feature-flags.md:45]: I looked at the original ticket, and this description seems misleading. I read this as thinking that users could access FF functionality without a license. But it looks like it was more of a UI issue -- you could see FF in the UI without a license but clicking it returned a 404.

**Review by douglas-j-bothwell — APPROVED** [review]
Approving this, but I think the FFM-8002 description really needs to be fixed.

**Pull request merged**